src/catch2/catch_message.hpp

```diff
@@ -110,7 +110,7 @@ namespace Catch {
 
 ///////////////////////////////////////////////////////////////////////////////
 #define INTERNAL_CATCH_UNSCOPED_INFO( macroName, log ) \
-    Catch::getResultCapture().emplaceUnscopedMessage( Catch::MessageBuilder( macroName##_catch_sr, CATCH_INTERNAL_LINEINFO, Catch::ResultWas::Info ) << log )
```

Comment:
This breaks streaming usage like

`UNSCOPED_INFO("Status is " << status.ok());`

I see that we do not have this test for `UNSCOPED_INFO`, only for `INFO` -> I assume that's why this change isn't on the `INFO` internals as well.

I added the test in 9c541ca7

Reply:
Okay, removed that part.